<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCCA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../IMAGENS/img.qrc">
    <normaloff>:/Tela principal/INSS.png</normaloff>:/Tela principal/INSS.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;Arial&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>Usuario Logado :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(208, 208, 208);</string>
        </property>
        <property name="text">
         <string>Cadastrar Usuário</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Data e Hora:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">image: url(:/Tela principal/INSS.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;Arial&quot;;
background-color: rgb(186, 186, 186);</string>
        </property>
        <property name="text">
         <string>TABELA DE DADOS :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <row>
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignJustify|AlignVCenter</set>
         </property>
        </row>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Número do Beneficio</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipo de Acontecimento</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fonte de Ação</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Espécie</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Número de Protocolo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome Segurado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CPF Segurado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NIT</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CEP</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rua</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bairro</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Estado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Municipio</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome Responsavel</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CPF \ CNPJ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor Dívida</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor Calculado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor de Entrada</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Parcelas Pagas</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total de Parcelas</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data da Cobrança</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data do Calculo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ação Regressiva</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDados">
    <property name="title">
     <string>Arquivos</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCadastrar"/>
    <addaction name="actionProcurar"/>
    <addaction name="actionEditar"/>
    <addaction name="actionApagar"/>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuDados"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCadastrar"/>
   <addaction name="actionProcurar"/>
   <addaction name="actionEditar"/>
   <addaction name="actionApagar"/>
   <addaction name="actionRefresh"/>
  </widget>
  <action name="actionCadastrar">
   <property name="icon">
    <iconset resource="../IMAGENS/img.qrc">
     <normaloff>:/Tela principal/Cadastro.png</normaloff>:/Tela principal/Cadastro.png</iconset>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar Beneficio</string>
   </property>
  </action>
  <action name="actionProcurar">
   <property name="icon">
    <iconset resource="../IMAGENS/img.qrc">
     <normaloff>:/Tela principal/Procurar.png</normaloff>:/Tela principal/Procurar.png</iconset>
   </property>
   <property name="text">
    <string>Procurar</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../IMAGENS/img.qrc">
     <normaloff>:/Tela principal/Refresh.png</normaloff>:/Tela principal/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="icon">
    <iconset resource="../IMAGENS/img.qrc">
     <normaloff>:/Tela principal/Editar.png</normaloff>:/Tela principal/Editar.png</iconset>
   </property>
   <property name="text">
    <string>Editar Dados</string>
   </property>
   <property name="toolTip">
    <string>Editar Dados</string>
   </property>
  </action>
  <action name="actionApagar">
   <property name="icon">
    <iconset resource="../IMAGENS/img.qrc">
     <normaloff>:/Tela principal/Apagar.png</normaloff>:/Tela principal/Apagar.png</iconset>
   </property>
   <property name="text">
    <string>Apagar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../IMAGENS/img.qrc"/>
 </resources>
 <connections/>
</ui>
